spec/controllers/people_controller_spec.rb

```diff
@@ -0,0 +1,159 @@
+      person = Person.create! valid_attributes
+      get :edit, {:id => person.to_param}, valid_session
+      expect(assigns(:person)).to eq(person)
+    end
+  end
+
+  describe "POST #create" do
+    context "with valid params" do
+      it "creates a new Person" do
+        expect {
+          post :create, {:person => valid_attributes}, valid_session
+        }.to change(Person, :count).by(1)
+      end
+
+      it "assigns a newly created person as @person" do
+        post :create, {:person => valid_attributes}, valid_session
+        expect(assigns(:person)).to be_a(Person)
+        expect(assigns(:person)).to be_persisted
+      end
+
+      it "redirects to the created person" do
+        post :create, {:person => valid_attributes}, valid_session
+        expect(response).to redirect_to(Person.last)
+      end
+    end
+
+    context "with invalid params" do
+      it "assigns a newly created but unsaved person as @person" do
+        post :create, {:person => invalid_attributes}, valid_session
+        expect(assigns(:person)).to be_a_new(Person)
+      end
+
+      it "re-renders the 'new' template" do
+        post :create, {:person => invalid_attributes}, valid_session
+        expect(response).to render_template("new")
+      end
+    end
+  end
+
+  describe "PUT #update" do
+    context "with valid params" do
+      let(:new_attributes) {
+        skip("Add a hash of attributes valid for your model")
+      }
+
+      it "updates the requested person" do
+        person = Person.create! valid_attributes
+        put :update, {:id => person.to_param, :person => new_attributes}, valid_session
+        person.reload
+        skip("Add assertions for updated state")
+      end
+
+      it "assigns the requested person as @person" do
+        person = Person.create! valid_attributes
+        put :update, {:id => person.to_param, :person => valid_attributes}, valid_session
+        expect(assigns(:person)).to eq(person)
+      end
+
+      it "redirects to the person" do
+        person = Person.create! valid_attributes
+        put :update, {:id => person.to_param, :person => valid_attributes}, valid_session
+        expect(response).to redirect_to(person)
+      end
+    end
+
+    context "with invalid params" do
+      it "assigns the person as @person" do
+        person = Person.create! valid_attributes
+        put :update, {:id => person.to_param, :person => invalid_attributes}, valid_session
+        expect(assigns(:person)).to eq(person)
+      end
+
+      it "re-renders the 'edit' template" do
+        person = Person.create! valid_attributes
+        put :update, {:id => person.to_param, :person => invalid_attributes}, valid_session
+        expect(response).to render_template("edit")
+      end
+    end
+  end
+
+  describe "DELETE #destroy" do
+    it "destroys the requested person" do
+      person = Person.create! valid_attributes
+      expect {
+        delete :destroy, {:id => person.to_param}, valid_session
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.